easybuild/easyconfigs/c/CGAL/CGAL-4.11-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,41 @@
+name = 'CGAL'
+version = '4.11'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-%(version)s']
+sources = [SOURCE_TAR_XZ]
+checksums = ['27a7762e5430f5392a1fe12a3a4abdfe667605c40224de1c6599f49d66cfbdd2']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('zlib', '1.2.11'),
+    ('Boost', '1.65.1'),
+    ('MPFR', '3.1.6'),
+    ('GMP', '6.1.2'),
+    ('libGLU', '9.0.0'),
+]
+
+builddependencies = [
+    ('CMake', '3.9.5'),
+    ('Eigen', '3.3.4', '', True),
+    ('Qt5', '5.9.2'),
+]
+
+configopts = "-DOpenGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
+configopts += "-DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.%s " % SHLIB_EXT
+configopts += "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s " % SHLIB_EXT
+configopts += "-DOpenGL_LIBRARIES=$EBROOTMESA/lib/libGL.%(ext)s\;EBROOTLIBGLU/lib/libGLU.%(ext)s " % {'ext': SHLIB_EXT}
+configopts += "-DWITH_ZLIB=ON -DWITH_MPFR=ON -DWITH_OpenGL=ON -DWITH_Eigen3=ON "
+configopts += "-DWITH_GMPXX=ON -DWITH_LAPACK=ON -DWITH_BLAS=ON "
+configopts += "-DBLAS_INCLUDE_DIR=$EBROOTOPENBLAS/include "
+configopts += "-DBLAS_LIBRARIES=$EBROOTOPENBLAS/lib/libopenblas.%s " % SHLIB_EXT
+configopts += "-DLAPACK_INCLUDE_DIR=$EBROOTSCALAPACK/include "
+configopts += "-DLAPACK_LIBRARIES=$EBROOTSCALAPACK/lib/libscalapack.a "
```

Comment:
There's a lot of stuff here that could/should be handled by the `CGAL` easyblock instead...